src/electron/customCSS.ts

```diff
@@ -0,0 +1,14 @@
+import { readFileSync } from 'fs';
+import { pathExistsSync } from 'fs-extra';
+import { userDataPath } from '../environment-remote';
+
+export default function loadCustomCSS() {
+  const path = userDataPath('config', 'custom.css');
```

Comment:
Please document this path and what this feature does in the README.md file for new users to try out if they choose to.

Reply:
Sure